assets/src/helpers/icon.tsx

```diff
@@ -149,6 +151,8 @@ export const CommuterRailIcon = svgIcon(commuterRailIconSvg)
 
 export const CrowdingIcon = svgIcon(crowdingIconSvg)
 
+export const DiamondTurnRightSvg = svgIcon(diamondTurnRightSvg)
```

Comment:
**suggestion:** there is actually a test suite for this file that does a basic render of each icon, you could add this one to the list if you want.